ovs/openflow.go

```diff
@@ -313,6 +313,13 @@ func (o *OpenFlowService) DumpFlows(bridge string) ([]*Flow, error) {
 	return o.DumpFlowsWithFlowArgs(bridge, nil)
 }
 
+// DumpMatchingFlows retrieves statistics of all matching flows for the specified bridge.
+// If a table has no active flows and has not been used for a lookup or matched
+// by an incoming packet, it is filtered from the output.
+func (o *OpenFlowService) DumpMatchingFlows(bridge string, flow *MatchFlow) ([]*Flow, error) {
```

Comment:
Do we need `DumpMatchingFlows`? Does it provide any value that `DumpFlowsWithFlowArgs` doesn't already provide?